nfafrontend/src/app/shared/data-storage.service.ts

```diff
@@ -3,8 +3,7 @@ import {Injectable} from '@angular/core';
 import {Http} from '@angular/http';
 import {Project} from './project.model';
 import {HttpParams} from '@angular/common/http';
-
-
+import {Type} from './type.model';
```

Comment:
I am still missing this file? Is it maybe part of git ignore or something?

Reply:
it is not the latest ... i still have 4  commits ahead ...could you accept this pull request in order to commit the rest?

Reply:
But those are 4 commits.
4 commits ahead means (afaik), once you merge, the master will have exactly 4 new commits (from you).
are you sure it is not the latest.

I would prefer if you closed the request, committed whatever is missing and make a new request. Otherwise we generate a broken master, again.